src/docs/index.less

```diff
@@ -37,6 +37,11 @@ html {
 		justify-content: space-between;
 		align-items: center;
 	}
+
+	&-props-table {
+		p { margin: 5px 0; }
+		td { padding: 2px 7px; }
```

Comment:
probably format these rule sets to match the format of other css rules sets in the file.